fe/fe-core/src/main/java/com/starrocks/qe/HDFSBackendSelector.java

```diff
@@ -227,44 +229,66 @@ private long computeTotalSize() {
 
     @Override
     public void computeScanRangeAssignment() throws UserException {
+        computeGeneralAssignment();
+        if (useIncrementalScanRanges) {
+            boolean hasMore = scanNode.hasMoreScanRanges();
+            TScanRangeParams end = new TScanRangeParams();
+            end.setScan_range(new TScanRange());
+            end.setPlaceholder(true);
+            end.setHas_more(hasMore);
+            for (ComputeNode computeNode : workerProvider.getAllWorkers()) {
+                assignment.put(computeNode.getId(), scanNode.getId().asInt(), end);
```

Comment:
If the assignment on this BE is not empty, do we also need to add an empty scan range?

Reply:
`end.setPlaceholder(true);`  marks scan range as a empty one. @Youngwb suggests to rename it as `is_empty`.

And this empty scan range will be handled in next PR: #50254